libafl/src/corpus/testcase.rs

```diff
@@ -4,6 +4,9 @@
 use alloc::string::String;
 use core::{default::Default, option::Option, time::Duration};
 
+#[cfg(feature = "std")]
+use std::{fs, path::Path};
```

Comment:
Is it okay for testcase.rs to depend on std?

Reply:
It needs to work without (for inmemory), but for storing to disk it needs std yes

Reply:
added set_Filename function for no_std.

Since there is no file creation/deletion, it just assigns value to struct.